docs/interfaces.md

```diff
@@ -138,6 +138,13 @@ owned by the user to read/write/lock.
 * Auto-Connect: yes on classic (traditional distributions), no otherwise
 * Transitional: yes
 
+### i2c
+
+Can access the registered i2c adapters in read-write mode. Suitable for
+talking to peripherial devices.
```

Comment:
Typo: should be 'peripheral'; that said, I think it best to remove the whole 'Suitable for...' and just leave the 'Can access...' sentence.